app/main.py

```diff
@@ -1,3 +1,65 @@
+from __future__ import division
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: float) -> None:
```

Comment:
Indicate in the type annotation that `km` could be an integer as well.

Reply:
Please consider comment above and make annotation as int | float